src/ci/pgo.sh

```diff
@@ -84,7 +85,7 @@ python3 ../x.py build --target=$PGO_HOST --host=$PGO_HOST \
     --rust-profile-generate=/tmp/rustc-pgo
 
 gather_profiles "Check,Debug,Opt" "All" \
-  "externs,ctfe-stress-4,inflate,cargo,token-stream-stress,match-stress-enum"
+  "externs,ctfe-stress-4,cargo-0.60.0,token-stream-stress,match-stress,tuple-stress"
```

Comment:
What's the distinction between this `gather_profiles` call and the one 20 lines above? The choice of benchmarks used for the one above seems much better, these are mostly secondary benchmarks.

Reply:
The first one gathers profiles for LLVM, the second one for `rustc` itself. I'm not sure what was the original idea, but I suppose that the first one should stress LLVM (so it focuses on `Debug`, `Opt` and real-world benchmarks) and the second one on the frontend (so it also includes `Check` and it includes various frontend stress tests). The benchmarks could be changed, but that's a neverending struggle. There is also a constraint that the profile gathering shouldn't take too long.

Reply:
Ok. I will r+, but if you have a chance to add comments about this before it gets merged, that would be nice.